src/utils.js

```diff
@@ -72,8 +74,16 @@ const version = () => {
 	console.log( `Wikipedia Preview - version ${APP_VERSION} (${GIT_HASH})` )
 }
 
+const getPreviewHtml = ( title, lang ) => {
+	return new Promise( ( resolve )=> {
+		requestPagePreview( lang, title, isTouch, data => {
```

Comment:
https://github.com/wikimedia/wikipedia-preview/blob/72b6f03bd6c9296de224a5807a13719ce37384b7/src/api.js#L68-L72

the third param should be callback on `requestPagePreview`

_interesting, it works without the problem previously_

Reply:
Whooa good catch, how was that working before? From the plugin side I was actually providing only three arguments, third one being the function, so I guess JS was smart enough to map the functions and omit the boolean?